README.md

```diff
@@ -8,129 +8,39 @@
 [![Docker Hub Downloads](https://img.shields.io/docker/pulls/cyberreboot/networkml.svg)](https://hub.docker.com/r/cyberreboot/networkml/)
 
 ## Overview
-NetworkML is the machine learning portion of our [Poseidon](https://github.com/CyberReboot/poseidon)
-project. The models in networkML answer two questions:
-  1. What is the role of the device in a particular packet capture (PCAP)?
-  2. Given that device's role, is that device acting properly or anomalously?
+NetworkML is the machine learning portion of our [Poseidon](https://github.com/CyberReboot/poseidon)project. The model in networkML classifies each device into a functional role via machine learning models trained on features derived from network traffic. "Functional role" refers to the authorized administrative purpose of the device on the network and includes roles as printer, mail server, and others typically found in an IT environment. Our internal analysis suggests networkML can achieve accuracy, precision, recall, and F1 scores in the high 90's when trained on devices from your own network. Whether this performance can transfer from IT environment to IT environment is an active area of our research.
```

Comment:
missing a space between the link and `project` here: `[Poseidon](https://github.com/CyberReboot/poseidon)project`

Reply:
Good catch. I corrected that mistake and one other small grammatical issue.